taskcluster/scripts/pipeline/train_taskcluster.py

```diff
@@ -77,7 +78,10 @@ def main(args):
             run_artifacts = set([os.path.basename(a["name"]) for a in resp.json()["artifacts"]])
 
             resumable = True
-            if run_artifacts.issuperset(CONTINUATION_ARTIFACTS):
+
+            if run_artifacts.issuperset(
+                CONTINUATION_ARTIFACTS.union({f"vocab.{src}.spm", f"vocab.{trg}.spm"})
```

Comment:
nit: Now that `CONTINUATION_ARTIFACTS` is no longer a complete set it might be good to add a comment around its original definition to indicate that. Or to move these two new entries to it, and call `.format()` on its items before checking `run_artifacts` against it.

Reply:
I'd prefer the .format() solution.